data/games/under_night_in_birth_exe_late_st/movespecs/enkidu.json

```diff
@@ -0,0 +1,152 @@
+{
+  "name": "Enkidu",
+  "movelists": [
+    [
+      "5A",
+      "2A",
+      "5B",
+      "2B",
+      "5C",
+      "5[C]",
+      "2C",
+      "2[C]",
+      "j.A",
+      "j.B",
+      "j.C"
+    ],
+    [
+      "5BB",
+      "5BBB",
+      "5CC",
+      "5CCC",
+      "2CC",
+      "2CCC",
+      "3C",
+      "3[C]",
+      "4C",
+      "6C",
+      "6[C]",
+      "j.CC"
+    ],
+    [
+      "66B",
+      "66C"
+    ],
+    [
+      {
+        "name": "Tidal Spin",
+        "inputs": [
+          "B+C"
+        ]
+      },
+      {
+        "name": "Throw",
+        "inputs": [
+          "A+D"
+        ]
+      },
+      {
+        "name": "Guard Cancel",
+        "inputs": [
+          "214D"
+        ]
+      },
+      {
+        "name": "Veil Off",
+        "inputs": [
+          "A+B+C"
+        ]
+      }
+    ],
+    [
+      {
+        "name": "First Prohibition",
```

Comment:
Let's also give First Prohibition and Second Prohibition `shortName` fields. The way follow ups are rendered you get some ugly word wrap on long names. Maybe "1st Pro" and "2nd Pro" or something?

Reply:
That makes sense.

I just noticed that in the spec, we have `follow_ups` and `shortName`. Was it intentional to use both camel case and underscores? Should we standardize on one or the other?

Reply:
I would like to standardize, I just haven't fixed `follow_ups` to `followUps` yet.

Reply:
I created #58 for tracking.